Program.cs

```diff
@@ -1,7 +1,26 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+using System.Text;
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {

```

Comment:
Fix the alignment issue mentioned in the Conversation.

Reply:
Done